Client/FeatureFlags/FeatureFlagsManager.swift

```diff
@@ -0,0 +1,78 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import Shared
+
+protocol FeatureFlagsProtocol { }
+
+extension FeatureFlagsProtocol {
+    var featureFlags: FeatureFlagsManager {
+        return FeatureFlagsManager.shared
+    }
+}
+
+/// An enum describing the featureID of all features, historical, avalable, or in development.
+enum FeatureFlagID: String, CaseIterable {
+    case chronologicalTabs
+    case inactiveTabs
+    case nimbus
+    case recentlySaved
+    case shakeToRestore
+}
```

Comment:
Is nimbus a feature too? 
Should we be calling this `FeatureFlagNames` 😅 

p.s. I am okay with Id as well

Reply:
We had it as a feature, yeah.

I'm fine with changing it to that. I'll update it. :)